src/utils/convert.js

```diff
@@ -83,20 +83,32 @@ ${props.join(',\n')}
 };
 
 const renderSampleField = (def) => {
-  const type = typesMap[def.type];
+  const type = typesMap[def.type] || 'string';
 
-  return `      ${def.key}: {
+  if (def.label) {
```

Comment:
JSON.dump seems like a good approach here, but I'd be careful with the big 1 liner - with all the parens and commas I find that very hard to grok on a skim.